.eslintrc.js

```diff
@@ -443,6 +469,66 @@ module.exports = {
               OLD_DUAL_LICENSE_HEADER,
               SAFER_LODASH_SET_HEADER,
               SAFER_LODASH_SET_LODASH_HEADER,
+              KBN_HANDLEBARS_HEADER,
+              KBN_HANDLEBARS_HANDLEBARS_HEADER,
+            ],
+          },
+        ],
+      },
+    },
+
+    /**
+     * @kbn/handlebars package requires special license headers
+     */
+    {
+      files: ['packages/kbn-handlebars/**/*.{js,mjs,ts,tsx}'],
+      rules: {
+        '@kbn/eslint/require-license-header': [
+          'error',
+          {
+            license: KBN_HANDLEBARS_HEADER,
+          },
+        ],
+        '@kbn/eslint/disallow-license-headers': [
+          'error',
+          {
+            licenses: [
+              APACHE_2_0_LICENSE_HEADER,
```

Comment:
Could we reduce this list to just the licenses that are in use in these files?

Reply:
The `@kbn/eslint/disallow-license-headers` section are all the licenses that are _not_ allowed in these files. The `@kbn/eslint/require-license-header` section above is the one that contains the license(s) that are required.

Reply:
Oh, sorry I missed this 😳